ludwig/schema/llms/peft.py

```diff
@@ -406,7 +406,9 @@ def description(cls) -> str:
 
 
 @DeveloperAPI
-@register_adapter("adaption_prompt")
+# TODO: <Alex>02/21/2024: Disabling AdaptionPrompt (waiting for PEFT release to fix
```

Comment:
Hey, can we create a GitHub issue and link it here? It's easier to find when we want to come back to it